spring-boot-project/spring-boot-test-autoconfigure/src/test/java/org/springframework/boot/test/autoconfigure/hateoas/HypermediaTestAutoConfigurationTests.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2012-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.test.autoconfigure.hateoas;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
```

Comment:
We don't use wildcard imports in Spring Boot's codebase. Building the project locally should show any problems like these as they're enforced by Checkstyle and Spring Java Format.

Reply:
;)